### PR TITLE
Document XTB/QCEngine incompatibility

### LINKED ISSUE #323: BespokeFit appears to ignore `--qc-compute-n-cores` with XTB
**Description and Reproduction**

```shell
micromamba run -n bespokefit openff-bespoke executor run    \
    --smiles "CCCCCC(=O)O"                                  \
    --force-field "openff-2.1.0.offxml"                     \
    --output-force-field "bespokefit-hexanoic-acid.offxml"  \
    --target-torsion "[#6X4:1]-[#6X3:2]=[#8X1+0]"           \
    --workflow "default"                                    \
    --qc-compute-n-cores 4                                  \
    --n-fragmenter-workers 1                                \
    --n-qc-compute-workers 1                                \
    --n-optimizer-workers 1                                 \
    --default-qc-spec xtb gfn2xtb none
```

This produces 24 threads on my 12 core/24 thread CPU during the QC step, and 1 thread on all other steps. I expect it to produce only 4 during QC, and 1 in other steps.

**Software versions**

- Which operating system and version are you using? Latest Arch, BespokeFit 0.2.3
- How did you install BespokeFit? `micromamba create -n bespokefit bespokefit xtb-python`
- Are you using Apple Silicon? If so, are you running BespokeFit in Rosetta (`osx-64`) or natively (`osx-arm64`)? Using AMD64
- What is the output of running `conda list`?

<details><summary>Output of <code>conda list</code></summary><pre>

List of packages in environment: "/home/joshmitchell/micromamba/envs/bespokefit"

  Name                           Version          Build                    Channel    
────────────────────────────────────────────────────────────────────────────────────────
  _libgcc_mutex                  0.1              conda_forge              conda-forge
  _openmp_mutex                  4.5              2_gnu                    conda-forge
  ambertools                     23.3             py311h9fea076_6          conda-forge
  amqp                           5.2.0            pyhd8ed1ab_1             conda-forge
  anyio                          4.2.0            pyhd8ed1ab_0             conda-forge
  argcomplete                    3.2.2            pyhd8ed1ab_0             conda-forge
  argon2-cffi                    23.1.0           pyhd8ed1ab_0             conda-forge
  argon2-cffi-bindings           21.2.0           py311h459d7ec_4          conda-forge
  arpack                         3.8.0            nompi_h0baa96a_101       conda-forge
  arrow                          1.3.0            pyhd8ed1ab_0             conda-forge
  asttokens                      2.4.1            pyhd8ed1ab_0             conda-forge
  astunparse                     1.6.3            pyhd8ed1ab_0             conda-forge
  async-lru                      2.0.4            pyhd8ed1ab_0             conda-forge
  async-timeout                  4.0.3            pyhd8ed1ab_0             conda-forge
  attrs                          23.2.0           pyh71513ae_0             conda-forge
  aws-c-auth                     0.7.15           h70caa3e_0               conda-forge
  aws-c-cal                      0.6.9            h14ec70c_3               conda-forge
  aws-c-common                   0.9.12           hd590300_0               conda-forge
  aws-c-compression              0.2.17           h572eabf_8               conda-forge
  aws-c-event-stream             0.4.1            h17cd1f3_5               conda-forge
  aws-c-http                     0.8.0            hc6da83f_5               conda-forge
  aws-c-io                       0.14.3           h3c8c088_1               conda-forge
  aws-c-mqtt                     0.10.1           h0ef3971_3               conda-forge
  aws-c-s3                       0.5.0            h1b46bed_2               conda-forge
  aws-c-sdkutils                 0.1.14           h572eabf_0               conda-forge
  aws-checksums                  0.1.17           h572eabf_7               conda-forge
  aws-crt-cpp                    0.26.1           h33f84b2_9               conda-forge
  aws-sdk-cpp                    1.11.242         h65f022c_0               conda-forge
  babel                          2.14.0           pyhd8ed1ab_0             conda-forge
  backports.zoneinfo             0.2.1            py311h38be061_8          conda-forge
  basis_set_exchange             0.9.1            pyhd8ed1ab_0             conda-forge
  beautifulsoup4                 4.12.3           pyha770c72_0             conda-forge
  billiard                       4.2.0            py311h459d7ec_0          conda-forge
  bleach                         6.1.0            pyhd8ed1ab_0             conda-forge
  blosc                          1.21.5           h0f2a231_0               conda-forge
  brotli                         1.1.0            hd590300_1               conda-forge
  brotli-bin                     1.1.0            hd590300_1               conda-forge
  brotli-python                  1.1.0            py311hb755f60_1          conda-forge
  bson                           0.5.9            py_0                     conda-forge
  bzip2                          1.0.8            hd590300_5               conda-forge
  c-ares                         1.26.0           hd590300_0               conda-forge
  c-blosc2                       2.13.2           hb4ffafa_0               conda-forge
  ca-certificates                2024.2.2         hbcca054_0               conda-forge
  cached-property                1.5.2            hd8ed1ab_1               conda-forge
  cached_property                1.5.2            pyha770c72_1             conda-forge
  cachetools                     5.3.2            pyhd8ed1ab_0             conda-forge
  cairo                          1.18.0           h3faef2a_0               conda-forge
  celery                         5.3.5            pyhd8ed1ab_0             conda-forge
  certifi                        2024.2.2         pyhd8ed1ab_0             conda-forge
  cffi                           1.16.0           py311hb3a22ac_0          conda-forge
  chardet                        5.2.0            py311h38be061_1          conda-forge
  charset-normalizer             3.3.2            pyhd8ed1ab_0             conda-forge
  chemper                        1.0.1            pyhd8ed1ab_0             conda-forge
  click                          8.1.7            unix_pyh707e725_0        conda-forge
  click-didyoumean               0.3.0            pyhd8ed1ab_0             conda-forge
  click-option-group             0.5.6            pyhd8ed1ab_0             conda-forge
  click-plugins                  1.1.1            py_0                     conda-forge
  click-repl                     0.3.0            pyhd8ed1ab_0             conda-forge
  colorama                       0.4.6            pyhd8ed1ab_0             conda-forge
  comm                           0.2.1            pyhd8ed1ab_0             conda-forge
  contourpy                      1.2.0            py311h9547e67_0          conda-forge
  cudatoolkit                    11.8.0           h4ba93d1_13              conda-forge
  cycler                         0.12.1           pyhd8ed1ab_0             conda-forge
  debugpy                        1.8.1            py311hb755f60_0          conda-forge
  decorator                      5.1.1            pyhd8ed1ab_0             conda-forge
  defusedxml                     0.7.1            pyhd8ed1ab_0             conda-forge
  entrypoints                    0.4              pyhd8ed1ab_0             conda-forge
  exceptiongroup                 1.2.0            pyhd8ed1ab_2             conda-forge
  executing                      2.0.1            pyhd8ed1ab_0             conda-forge
  expat                          2.5.0            hcb278e6_1               conda-forge
  fastapi                        0.86.0           pyhd8ed1ab_0             conda-forge
  fftw                           3.3.10           nompi_hc118613_108       conda-forge
  font-ttf-dejavu-sans-mono      2.37             hab24e00_0               conda-forge
  font-ttf-inconsolata           3.000            h77eed37_0               conda-forge
  font-ttf-source-code-pro       2.038            h77eed37_0               conda-forge
  font-ttf-ubuntu                0.83             h77eed37_1               conda-forge
  fontconfig                     2.14.2           h14ed4e7_0               conda-forge
  fonts-conda-ecosystem          1                0                        conda-forge
  fonts-conda-forge              1                0                        conda-forge
  fonttools                      4.49.0           py311h459d7ec_0          conda-forge
  forcebalance                   1.9.6            py311h92ebd52_1          conda-forge
  fqdn                           1.5.1            pyhd8ed1ab_0             conda-forge
  freetype                       2.12.1           h267a509_2               conda-forge
  freetype-py                    2.3.0            pyhd8ed1ab_0             conda-forge
  future                         0.18.3           pyhd8ed1ab_0             conda-forge
  geometric                      1.0              pyhd8ed1ab_1             conda-forge
  gettext                        0.21.1           h27087fc_0               conda-forge
  gflags                         2.2.2            he1b5a44_1004            conda-forge
  glog                           0.6.0            h6f12383_0               conda-forge
  greenlet                       3.0.3            py311hb755f60_0          conda-forge
  h11                            0.14.0           pyhd8ed1ab_0             conda-forge
  h2                             4.1.0            pyhd8ed1ab_0             conda-forge
  h5py                           3.10.0           nompi_py311hebc2b07_101  conda-forge
  hdf4                           4.2.15           h2a13503_7               conda-forge
  hdf5                           1.14.3           nompi_h4f84152_100       conda-forge
  hpack                          4.0.0            pyh9f0ad1d_0             conda-forge
  httpcore                       1.0.3            pyhd8ed1ab_0             conda-forge
  httpx                          0.26.0           pyhd8ed1ab_0             conda-forge
  hyperframe                     6.0.1            pyhd8ed1ab_0             conda-forge
  icu                            73.2             h59595ed_0               conda-forge
  idna                           3.6              pyhd8ed1ab_0             conda-forge
  importlib-metadata             7.0.1            pyha770c72_0             conda-forge
  importlib_metadata             7.0.1            hd8ed1ab_0               conda-forge
  importlib_resources            6.1.1            pyhd8ed1ab_0             conda-forge
  ipykernel                      6.29.2           pyhd33586a_0             conda-forge
  ipython                        8.21.0           pyh707e725_0             conda-forge
  ipywidgets                     8.1.2            pyhd8ed1ab_0             conda-forge
  isoduration                    20.11.0          pyhd8ed1ab_0             conda-forge
  jedi                           0.19.1           pyhd8ed1ab_0             conda-forge
  jinja2                         3.1.3            pyhd8ed1ab_0             conda-forge
  joblib                         1.3.2            pyhd8ed1ab_0             conda-forge
  json5                          0.9.14           pyhd8ed1ab_0             conda-forge
  jsonpointer                    2.4              py311h38be061_3          conda-forge
  jsonschema                     4.21.1           pyhd8ed1ab_0             conda-forge
  jsonschema-specifications      2023.12.1        pyhd8ed1ab_0             conda-forge
  jsonschema-with-format-nongpl  4.21.1           pyhd8ed1ab_0             conda-forge
  jupyter-lsp                    2.2.2            pyhd8ed1ab_0             conda-forge
  jupyter_client                 8.6.0            pyhd8ed1ab_0             conda-forge
  jupyter_core                   5.7.1            py311h38be061_0          conda-forge
  jupyter_events                 0.9.0            pyhd8ed1ab_0             conda-forge
  jupyter_server                 2.12.5           pyhd8ed1ab_0             conda-forge
  jupyter_server_terminals       0.5.2            pyhd8ed1ab_0             conda-forge
  jupyterlab                     4.1.1            pyhd8ed1ab_0             conda-forge
  jupyterlab_pygments            0.3.0            pyhd8ed1ab_1             conda-forge
  jupyterlab_server              2.25.3           pyhd8ed1ab_0             conda-forge
  jupyterlab_widgets             3.0.10           pyhd8ed1ab_0             conda-forge
  keyutils                       1.6.1            h166bdaf_0               conda-forge
  kiwisolver                     1.4.5            py311h9547e67_1          conda-forge
  kombu                          5.3.3            py311h38be061_0          conda-forge
  krb5                           1.21.2           h659d440_0               conda-forge
  lcms2                          2.16             hb7c19ff_0               conda-forge
  ld_impl_linux-64               2.40             h41732ed_0               conda-forge
  lerc                           4.0.0            h27087fc_0               conda-forge
  libabseil                      20230802.1       cxx17_h59595ed_0         conda-forge
  libaec                         1.1.2            h59595ed_1               conda-forge
  libarrow                       15.0.0           he2c5238_2_cpu           conda-forge
  libarrow-acero                 15.0.0           h59595ed_2_cpu           conda-forge
  libarrow-dataset               15.0.0           h59595ed_2_cpu           conda-forge
  libarrow-flight                15.0.0           hdc44a87_2_cpu           conda-forge
  libarrow-flight-sql            15.0.0           hfbc7f12_2_cpu           conda-forge
  libarrow-gandiva               15.0.0           hacb8726_2_cpu           conda-forge
  libarrow-substrait             15.0.0           hfbc7f12_2_cpu           conda-forge
  libblas                        3.9.0            21_linux64_openblas      conda-forge
  libboost                       1.82.0           h6fcfa73_6               conda-forge
  libboost-python                1.82.0           py311h92ebd52_6          conda-forge
  libbrotlicommon                1.1.0            hd590300_1               conda-forge
  libbrotlidec                   1.1.0            hd590300_1               conda-forge
  libbrotlienc                   1.1.0            hd590300_1               conda-forge
  libcblas                       3.9.0            21_linux64_openblas      conda-forge
  libcrc32c                      1.1.2            h9c3ff4c_0               conda-forge
  libcurl                        8.5.0            hca28451_0               conda-forge
  libdeflate                     1.19             hd590300_0               conda-forge
  libedit                        3.1.20191231     he28a2e2_2               conda-forge
  libev                          4.33             hd590300_2               conda-forge
  libevent                       2.1.12           hf998b51_1               conda-forge
  libexpat                       2.5.0            hcb278e6_1               conda-forge
  libffi                         3.4.2            h7f98852_5               conda-forge
  libgcc-ng                      13.2.0           h807b86a_5               conda-forge
  libgfortran-ng                 13.2.0           h69a702a_5               conda-forge
  libgfortran5                   13.2.0           ha4646dd_5               conda-forge
  libglib                        2.78.3           h783c2da_0               conda-forge
  libgomp                        13.2.0           h807b86a_5               conda-forge
  libgoogle-cloud                2.12.0           hef10d8f_5               conda-forge
  libgrpc                        1.60.1           h74775cd_0               conda-forge
  libiconv                       1.17             hd590300_2               conda-forge
  libjpeg-turbo                  3.0.0            hd590300_1               conda-forge
  liblapack                      3.9.0            21_linux64_openblas      conda-forge
  libllvm15                      15.0.7           hb3ce162_4               conda-forge
  libnetcdf                      4.9.2            nompi_h9612171_113       conda-forge
  libnghttp2                     1.58.0           h47da74e_1               conda-forge
  libnl                          3.9.0            hd590300_0               conda-forge
  libnsl                         2.0.1            hd590300_0               conda-forge
  libnuma                        2.0.16           h0b41bf4_1               conda-forge
  libopenblas                    0.3.26           pthreads_h413a1c8_0      conda-forge
  libparquet                     15.0.0           h352af49_2_cpu           conda-forge
  libpng                         1.6.42           h2797004_0               conda-forge
  libprotobuf                    4.25.1           hf27288f_1               conda-forge
  libre2-11                      2023.06.02       h7a70373_0               conda-forge
  libsodium                      1.0.18           h36c2ea0_1               conda-forge
  libsqlite                      3.45.1           h2797004_0               conda-forge
  libssh2                        1.11.0           h0841786_0               conda-forge
  libstdcxx-ng                   13.2.0           h7e041cc_5               conda-forge
  libthrift                      0.19.0           hb90f79a_1               conda-forge
  libtiff                        4.6.0            ha9c0a0a_2               conda-forge
  libutf8proc                    2.8.0            h166bdaf_0               conda-forge
  libuuid                        2.38.1           h0b41bf4_0               conda-forge
  libwebp-base                   1.3.2            hd590300_0               conda-forge
  libxcb                         1.15             h0b41bf4_0               conda-forge
  libxcrypt                      4.4.36           hd590300_1               conda-forge
  libxml2                        2.12.5           h232c23b_0               conda-forge
  libxslt                        1.1.39           h76b75d6_0               conda-forge
  libzip                         1.10.1           h2629f0a_3               conda-forge
  libzlib                        1.2.13           hd590300_5               conda-forge
  lxml                           5.1.0            py311h9691dec_0          conda-forge
  lz4-c                          1.9.4            hcb278e6_0               conda-forge
  lzo                            2.10             h516909a_1000            conda-forge
  markdown-it-py                 3.0.0            pyhd8ed1ab_0             conda-forge
  markupsafe                     2.1.5            py311h459d7ec_0          conda-forge
  matplotlib-base                3.8.3            py311h54ef318_0          conda-forge
  matplotlib-inline              0.1.6            pyhd8ed1ab_0             conda-forge
  mctc-lib                       0.3.1            h74f4db8_0               conda-forge
  mda-xdrlib                     0.2.0            pyhd8ed1ab_0             conda-forge
  mdtraj                         1.9.9            py311h90fe790_1          conda-forge
  mdurl                          0.1.2            pyhd8ed1ab_0             conda-forge
  mistune                        3.0.2            pyhd8ed1ab_0             conda-forge
  msgpack-python                 1.0.7            py311h9547e67_0          conda-forge
  munkres                        1.1.4            pyh9f0ad1d_0             conda-forge
  nbclient                       0.8.0            pyhd8ed1ab_0             conda-forge
  nbconvert-core                 7.16.0           pyhd8ed1ab_0             conda-forge
  nbformat                       5.9.2            pyhd8ed1ab_0             conda-forge
  ncurses                        6.4              h59595ed_2               conda-forge
  nest-asyncio                   1.6.0            pyhd8ed1ab_0             conda-forge
  netcdf-fortran                 4.6.1            nompi_hacb5139_103       conda-forge
  networkx                       3.2.1            pyhd8ed1ab_0             conda-forge
  nglview                        3.1.1            pyh15ce09e_0             conda-forge
  nomkl                          1.0              h5ca1d4c_0               conda-forge
  notebook                       7.1.0            pyhd8ed1ab_0             conda-forge
  notebook-shim                  0.2.4            pyhd8ed1ab_0             conda-forge
  numexpr                        2.9.0            py311h039bad6_100        conda-forge
  numpy                          1.26.4           py311h64a7726_0          conda-forge
  ocl-icd                        2.3.2            hd590300_0               conda-forge
  ocl-icd-system                 1.0.0            1                        conda-forge
  openff-amber-ff-ports          0.0.4            pyhca7485f_0             conda-forge
  openff-bespokefit              0.2.3            pyhd8ed1ab_1             conda-forge
  openff-forcefields             2024.01.0        pyhca7485f_0             conda-forge
  openff-fragmenter-base         0.2.1            pyhd8ed1ab_0             conda-forge
  openff-interchange             0.3.18           pyhd8ed1ab_0             conda-forge
  openff-interchange-base        0.3.18           pyhd8ed1ab_0             conda-forge
  openff-models                  0.1.2            pyhca7485f_0             conda-forge
  openff-qcsubmit                0.5.0            pyhd8ed1ab_0             conda-forge
  openff-toolkit                 0.14.5           pyhd8ed1ab_1             conda-forge
  openff-toolkit-base            0.14.5           pyhd8ed1ab_1             conda-forge
  openff-units                   0.2.1            pyh1a96a4e_0             conda-forge
  openff-utilities               0.1.12           pyhd8ed1ab_0             conda-forge
  openjpeg                       2.5.0            h488ebb8_3               conda-forge
  openmm                         8.1.1            py311h6d2dbb8_1          conda-forge
  openssl                        3.2.1            hd590300_0               conda-forge
  orc                            1.9.2            h7829240_1               conda-forge
  overrides                      7.7.0            pyhd8ed1ab_0             conda-forge
  packaging                      23.2             pyhd8ed1ab_0             conda-forge
  packmol                        20.010           h86c2bf4_0               conda-forge
  pandas                         2.2.0            py311h320fe9a_0          conda-forge
  pandocfilters                  1.5.0            pyhd8ed1ab_0             conda-forge
  panedr                         0.8.0            pyhd8ed1ab_0             conda-forge
  parmed                         4.2.2            py311hb755f60_1          conda-forge
  parso                          0.8.3            pyhd8ed1ab_0             conda-forge
  pcre2                          10.42            hcad00b1_0               conda-forge
  perl                           5.32.1           7_hd590300_perl5         conda-forge
  pexpect                        4.9.0            pyhd8ed1ab_0             conda-forge
  pickleshare                    0.7.5            py_1003                  conda-forge
  pillow                         10.2.0           py311ha6c5da5_0          conda-forge
  pint                           0.21             pyhd8ed1ab_0             conda-forge
  pip                            24.0             pyhd8ed1ab_0             conda-forge
  pixman                         0.43.2           h59595ed_0               conda-forge
  pkgutil-resolve-name           1.3.10           pyhd8ed1ab_1             conda-forge
  platformdirs                   4.2.0            pyhd8ed1ab_0             conda-forge
  plotly                         5.19.0           pyhd8ed1ab_0             conda-forge
  prometheus_client              0.20.0           pyhd8ed1ab_0             conda-forge
  prompt-toolkit                 3.0.42           pyha770c72_0             conda-forge
  prompt_toolkit                 3.0.42           hd8ed1ab_0               conda-forge
  psutil                         5.9.8            py311h459d7ec_0          conda-forge
  pthread-stubs                  0.4              h36c2ea0_1001            conda-forge
  ptyprocess                     0.7.0            pyhd3deb0d_0             conda-forge
  pure_eval                      0.2.2            pyhd8ed1ab_0             conda-forge
  py-cpuinfo                     9.0.0            pyhd8ed1ab_0             conda-forge
  pyarrow                        15.0.0           py311h39c9aba_2_cpu      conda-forge
  pycairo                        1.26.0           py311h8feb60e_0          conda-forge
  pycparser                      2.21             pyhd8ed1ab_0             conda-forge
  pydantic                       1.10.13          py311h459d7ec_1          conda-forge
  pyedr                          0.8.0            pyhd8ed1ab_0             conda-forge
  pygments                       2.17.2           pyhd8ed1ab_0             conda-forge
  pymbar                         3.1.1            py311h4c7f6c3_2          conda-forge
  pyparsing                      3.1.1            pyhd8ed1ab_0             conda-forge
  pysocks                        1.7.1            pyha2e5f31_6             conda-forge
  pytables                       3.9.2            py311h10c7f7f_1          conda-forge
  python                         3.11.8           hab00c5b_0_cpython       conda-forge
  python-constraint              1.4.0            py_0                     conda-forge
  python-dateutil                2.8.2            pyhd8ed1ab_0             conda-forge
  python-fastjsonschema          2.19.1           pyhd8ed1ab_0             conda-forge
  python-json-logger             2.0.7            pyhd8ed1ab_0             conda-forge
  python-tzdata                  2024.1           pyhd8ed1ab_0             conda-forge
  python_abi                     3.11             4_cp311                  conda-forge
  pytz                           2024.1           pyhd8ed1ab_0             conda-forge
  pyyaml                         6.0.1            py311h459d7ec_1          conda-forge
  pyzmq                          25.1.2           py311h34ded2d_0          conda-forge
  qcelemental                    0.27.1           pyhd8ed1ab_0             conda-forge
  qcengine                       0.29.0           pyhd8ed1ab_0             conda-forge
  qcportal                       0.15.8           pyhd8ed1ab_0             conda-forge
  rdkit                          2023.09.5        py311h4c2f14b_0          conda-forge
  rdma-core                      50.0             hd3aeb46_0               conda-forge
  re2                            2023.06.02       h2873b5e_0               conda-forge
  readline                       8.2              h8228510_1               conda-forge
  redis-py                       5.0.1            pyhd8ed1ab_0             conda-forge
  redis-server                   7.2.4            h06160fa_0               conda-forge
  referencing                    0.33.0           pyhd8ed1ab_0             conda-forge
  regex                          2023.12.25       py311h459d7ec_0          conda-forge
  reportlab                      4.1.0            py311h459d7ec_0          conda-forge
  requests                       2.31.0           pyhd8ed1ab_0             conda-forge
  rfc3339-validator              0.1.4            pyhd8ed1ab_0             conda-forge
  rfc3986-validator              0.1.1            pyh9f0ad1d_0             conda-forge
  rich                           13.7.0           pyhd8ed1ab_0             conda-forge
  rlpycairo                      0.2.0            pyhd8ed1ab_0             conda-forge
  rpds-py                        0.18.0           py311h46250e7_0          conda-forge
  s2n                            1.4.3            h06160fa_0               conda-forge
  scipy                          1.12.0           py311h64a7726_2          conda-forge
  send2trash                     1.8.2            pyh41d4057_0             conda-forge
  setuptools                     69.0.3           pyhd8ed1ab_0             conda-forge
  six                            1.16.0           pyh6c4a22f_0             conda-forge
  smirnoff99frosst               1.1.0            pyh44b312d_0             conda-forge
  snappy                         1.1.10           h9fff704_0               conda-forge
  sniffio                        1.3.0            pyhd8ed1ab_0             conda-forge
  soupsieve                      2.5              pyhd8ed1ab_1             conda-forge
  sqlalchemy                     2.0.27           py311h459d7ec_0          conda-forge
  stack_data                     0.6.2            pyhd8ed1ab_0             conda-forge
  starlette                      0.20.4           pyhd8ed1ab_1             conda-forge
  tenacity                       8.2.3            pyhd8ed1ab_0             conda-forge
  terminado                      0.18.0           pyh0d859eb_0             conda-forge
  tinycss2                       1.2.1            pyhd8ed1ab_0             conda-forge
  tk                             8.6.13           noxft_h4845f30_101       conda-forge
  tomli                          2.0.1            pyhd8ed1ab_0             conda-forge
  tornado                        6.3.3            py311h459d7ec_1          conda-forge
  torsiondrive                   1.1.0            pyhd8ed1ab_0             conda-forge
  tqdm                           4.66.2           pyhd8ed1ab_0             conda-forge
  traitlets                      5.14.1           pyhd8ed1ab_0             conda-forge
  types-python-dateutil          2.8.19.20240106  pyhd8ed1ab_0             conda-forge
  typing-extensions              4.9.0            hd8ed1ab_0               conda-forge
  typing_extensions              4.9.0            pyha770c72_0             conda-forge
  typing_utils                   0.1.0            pyhd8ed1ab_0             conda-forge
  tzdata                         2024a            h0c530f3_0               conda-forge
  ucx                            1.15.0           h75e419f_3               conda-forge
  unidecode                      1.3.8            pyhd8ed1ab_0             conda-forge
  uri-template                   1.3.0            pyhd8ed1ab_0             conda-forge
  urllib3                        2.2.1            pyhd8ed1ab_0             conda-forge
  uvicorn                        0.27.1           py311h38be061_0          conda-forge
  vine                           5.1.0            pyhd8ed1ab_0             conda-forge
  wcwidth                        0.2.13           pyhd8ed1ab_0             conda-forge
  webcolors                      1.13             pyhd8ed1ab_0             conda-forge
  webencodings                   0.5.1            pyhd8ed1ab_2             conda-forge
  websocket-client               1.7.0            pyhd8ed1ab_0             conda-forge
  wheel                          0.42.0           pyhd8ed1ab_0             conda-forge
  widgetsnbextension             4.0.10           pyhd8ed1ab_0             conda-forge
  xmltodict                      0.13.0           pyhd8ed1ab_0             conda-forge
  xorg-kbproto                   1.0.7            h7f98852_1002            conda-forge
  xorg-libice                    1.1.1            hd590300_0               conda-forge
  xorg-libsm                     1.2.4            h7391055_0               conda-forge
  xorg-libx11                    1.8.7            h8ee46fc_0               conda-forge
  xorg-libxau                    1.0.11           hd590300_0               conda-forge
  xorg-libxdmcp                  1.1.3            h7f98852_0               conda-forge
  xorg-libxext                   1.3.4            h0b41bf4_2               conda-forge
  xorg-libxrender                0.9.11           hd590300_0               conda-forge
  xorg-libxt                     1.3.0            hd590300_1               conda-forge
  xorg-renderproto               0.11.1           h7f98852_1002            conda-forge
  xorg-xextproto                 7.3.0            h0b41bf4_1003            conda-forge
  xorg-xproto                    7.0.31           h7f98852_1007            conda-forge
  xtb                            6.5.1            h03160e7_1               conda-forge
  xtb-python                     22.1             py311h2582759_0          conda-forge
  xz                             5.2.6            h166bdaf_0               conda-forge
  yaml                           0.2.5            h7f98852_2               conda-forge
  zeromq                         4.3.5            h59595ed_0               conda-forge
  zipp                           3.17.0           pyhd8ed1ab_0             conda-forge
  zlib                           1.2.13           hd590300_5               conda-forge
  zlib-ng                        2.0.7            h0b41bf4_0               conda-forge
  zstd                           1.5.5            hfc55251_0               conda-forge

</pre></details>

### PR DESCRIPTION
**Pull request by mattwthompson**
Resolves #323

### CODE REVIEW

File: docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ openff-bespoke executor run --file                 "acetaminophen.sdf" \
                             --default-qc-spec xtb gfn2xtb none
 ```
 
+:::{note}
+For XTB (only), `--qc-compute-n-cores` is ignored because of miscommunications between QCEngine and XTB, but this can be worked around by setting the environment variable `OMP_NUM_THREADS`. See the FAQ for details.
+:::
```
Comment (mattwthompson) [inline docs/getting-started/quick-start.md:106]: A direct link would be nice but I chose not to dedicate the time to figuring that out. It should remain a single click away in the rendered docs

**Review by j-wags — APPROVED** [review]
Excellent - Thanks for the quick action @mattwthompson.

**Comment by j-wags** [comment]
Oh, and did anyone sanity check that this works locally? I'm doing so now but I'll take someone's word if they've already done it.

**Comment by j-wags** [comment]
Ok, I verified this works locally. Merging!

**Comment by mattwthompson** [comment]
Whoops, sniped

**Pull request merged**